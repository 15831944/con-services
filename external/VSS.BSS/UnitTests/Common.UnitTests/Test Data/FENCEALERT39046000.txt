<?xml version="1.0" encoding="UTF-8"?> 
<fence:topic xmlns:fence="http://www.cat.com/em/feed/v5/Fence" xmlns:feed="http://www.cat.com/em/feed/v5/Common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cat.com/em/feed/v5/Fence https://emfeed.rd.cat.com/EMFeedFiles/schema/v5/Fence.xsd"> 
    
<nextBuffer> 
   <url>https://emfeed.rd.cat.com/EMFeedService/v5/feed/allqueue/FENCEALERT/33635473</url> 
   <moreData>false</moreData> 
</nextBuffer> 
<message> 
      <header>
         <messageId>25636396</messageId>
         <masterMsgId>31484216</masterMsgId>
         <equipment>
            <serialNumber>-B9D01487</serialNumber>
            <make>CAT</make>
            <model>140M</model>
         </equipment>
         <moduleCode>-DQCAT0009325Q1</moduleCode>
         <moduleTime>2009-10-04 04:57:47</moduleTime>
         <receivedTime>2009-10-04 05:00:04</receivedTime>
      </header>
      <timeWatchActive>Y</timeWatchActive> 
      <exclusiveWatchActive>Y</exclusiveWatchActive> 
      <inclusiveWatchActive>Y</inclusiveWatchActive> 
      <timeWatchAlarm>Y</timeWatchAlarm> 
      <exclusiveWatchAlarm>N</exclusiveWatchAlarm> 
      <inclusiveWatchAlarm>N</inclusiveWatchAlarm> 
   </message> 
<message> 
      <header>
         <messageId>25636396</messageId>
         <masterMsgId>31484216</masterMsgId>
         <equipment>
            <serialNumber>-B9D01487</serialNumber>
            <make>CAT</make>
            <model>140M</model>
         </equipment>
         <moduleCode>-DQCAT0009325Q1</moduleCode>
         <moduleTime>2009-10-04 04:57:47</moduleTime>
         <receivedTime>2009-10-04 05:00:04</receivedTime>
      </header>
      <timeWatchActive>Y</timeWatchActive> 
      <exclusiveWatchActive>Y</exclusiveWatchActive> 
      <inclusiveWatchActive>Y</inclusiveWatchActive> 
      <timeWatchAlarm>Y</timeWatchAlarm> 
      <exclusiveWatchAlarm>Y</exclusiveWatchAlarm> 
      <inclusiveWatchAlarm>Y</inclusiveWatchAlarm> 
      <satelliteBlockage>Y</satelliteBlockage> 
      <disconnectSwitchUsed>Y</disconnectSwitchUsed> 
   </message> 
</fence:topic> 
