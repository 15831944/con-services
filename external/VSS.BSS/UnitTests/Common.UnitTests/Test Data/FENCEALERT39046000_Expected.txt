<?xml version="1.0" encoding="utf-8"?><ArrayOfNHDataWrapper xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.nighthawk.com/nighthawk/service/NHData/2009/10"><NHDataWrapper><Data z:Id="i1" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataFenceAlarm" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>1</d3p1:AssetID><d3p1:DebugRefID>25636396</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:DisconnectSwitchUsed>false</d3p1:DisconnectSwitchUsed><d3p1:EventUTC>2009-10-04T04:57:47</d3p1:EventUTC><d3p1:ExclusiveWatchActive>true</d3p1:ExclusiveWatchActive><d3p1:ExclusiveWatchAlarm>false</d3p1:ExclusiveWatchAlarm><d3p1:GPSDeviceID>-DQCAT0009325Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InclusiveWatchActive>true</d3p1:InclusiveWatchActive><d3p1:InclusiveWatchAlarm>false</d3p1:InclusiveWatchAlarm><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:SatelliteBlockage>false</d3p1:SatelliteBlockage><d3p1:SourceMsgID>31484216</d3p1:SourceMsgID><d3p1:TimeWatchActive>true</d3p1:TimeWatchActive><d3p1:TimeWatchAlarm>true</d3p1:TimeWatchAlarm><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper><NHDataWrapper><Data z:Id="i2" xmlns:d3p1="http://nighthawk.vss.com/data/2009/03" i:type="d3p1:DataFenceAlarm" xmlns:z="http://schemas.microsoft.com/2003/10/Serialization/"><EntityKey xmlns:d4p1="http://schemas.datacontract.org/2004/07/System.Data" i:nil="true" xmlns="http://schemas.datacontract.org/2004/07/System.Data.Objects.DataClasses" /><d3p1:AssetID>1</d3p1:AssetID><d3p1:DebugRefID>25636396</d3p1:DebugRefID><d3p1:DeviceType>ProductLink</d3p1:DeviceType><d3p1:DisconnectSwitchUsed>true</d3p1:DisconnectSwitchUsed><d3p1:EventUTC>2009-10-04T04:57:47</d3p1:EventUTC><d3p1:ExclusiveWatchActive>true</d3p1:ExclusiveWatchActive><d3p1:ExclusiveWatchAlarm>true</d3p1:ExclusiveWatchAlarm><d3p1:GPSDeviceID>-DQCAT0009325Q1</d3p1:GPSDeviceID><d3p1:ID>0</d3p1:ID><d3p1:InclusiveWatchActive>true</d3p1:InclusiveWatchActive><d3p1:InclusiveWatchAlarm>true</d3p1:InclusiveWatchAlarm><d3p1:InsertUTC>0001-01-01T00:00:00</d3p1:InsertUTC><d3p1:SatelliteBlockage>true</d3p1:SatelliteBlockage><d3p1:SourceMsgID>31484216</d3p1:SourceMsgID><d3p1:TimeWatchActive>true</d3p1:TimeWatchActive><d3p1:TimeWatchAlarm>true</d3p1:TimeWatchAlarm><d3p1:fk_DimSourceID>1</d3p1:fk_DimSourceID></Data><StoreAttempts>0</StoreAttempts></NHDataWrapper></ArrayOfNHDataWrapper>